src/redux/books/books.js

```diff
@@ -0,0 +1,22 @@
+const ADD   = 'bookstore/book/ADD';
+const REMOVE = 'bookstore/book/REMOVE';
+
+const state = [];
+
+export const addBook = (book) => {
+  return { type: ADD, book };
+}
+
+export const removeBook = (book) => {
+  return { type: REMOVE, book };
+}
+
+const bookReducer = (state, action) => {
+  switch (action.type) {
+    case ADD: return [...state, action.book];
+    case REMOVE: return state.filter((item => item.id === action.book.id));
```

Comment:
- [ ] Right now, I am not able to add or delete books from your app, so I think there must be some changes to apply yet.
- [ ] I am not sure if this function will work as the project asks because in order to delete a book you are checking for a book that has the same id as the action book id and you are filtering through that, which makes me think is not correct. Maybe, you want to do something like `return state.filter(item => item.id !== action.book.id`. Realize about the `!==` instead of `===`. I am not able to test this in your app right now because of the error I mentioned earlier, but take this into account.